### PR TITLE
drivers: wifi: Clean boot logs

### PR DESCRIPTION
**Pull request by sachinthegreen**
1. Clean up Wi-Fi driver boot log messages by removing non-essential informational messages 
2. Display firmware version on successful boot of Sheliak

This addresses the following jira issues:
[SHEL-1347]

### CODE REVIEW

File: drivers/wifi/nrf700x/osal/fw_if/umac_if/src/cmd.c [hunk 1]
```diff
@@ -136,9 +136,7 @@ enum wifi_nrf_status umac_cmd_init(struct wifi_nrf_fmac_dev_ctx *fmac_dev_ctx,
 #ifdef CONFIG_NRF700X_TCP_IP_CHECKSUM_OFFLOAD
 	umac_cmd_data->sys_params.tcp_ip_checksum_offload = 1;
 #endif /* CONFIG_NRF700X_TCP_IP_CHECKSUM_OFFLOAD */
-	wifi_nrf_osal_log_info(fmac_dev_ctx->fpriv->opriv, "RPU LPM type: %s\n",
-		umac_cmd_data->sys_params.sleep_enable == 2 ? "HW" :
-		umac_cmd_data->sys_params.sleep_enable == 1 ? "SW" : "DISABLED");
+
```
Comment (krish2718) [inline drivers/wifi/nrf700x/osal/fw_if/umac_if/src/cmd.c:139]: This is handy, so, should be converted to `_dbg`

**Review by krish2718 — CHANGES_REQUESTED** [review]
IMHO, we should convert those prints to _dbg rather than removing them.

File: drivers/wifi/nrf700x/zephyr/src/zephyr_fmac_main.c [hunk 2]
```diff
@@ -312,6 +314,25 @@ enum wifi_nrf_status wifi_nrf_fmac_dev_add_zep(struct wifi_nrf_drv_priv_zep *drv
...
+	status = wifi_nrf_fmac_ver_get(rpu_ctx,
+				       &umac_ver,
+				       &lmac_ver);
+
+	if (status != WIFI_NRF_STATUS_SUCCESS) {
+		LOG_ERR("%s: FW version read failed\n", __func__);
+		goto out;
+	}
+
+	LOG_INF("Firmware (Version %d:%d:%d:%d / %d:%d:%d:%d) booted succesfully\n",
```
[the remark this answers is not in the document]
Comment (sachinthegreen) [reply drivers/wifi/nrf700x/zephyr/src/zephyr_fmac_main.c:326]: @udaynordic : WDYT?

**Review by krish2718 — APPROVED** [review]

File: drivers/wifi/nrf700x/zephyr/src/zephyr_fmac_main.c (hunk 2 above)
[the remark this answers is not in the document]
Comment (udaynordic) [reply drivers/wifi/nrf700x/zephyr/src/zephyr_fmac_main.c]: I would prefer to have this as info than debug.

**Review by udaynordic — APPROVED** [review]

**Review by rado17 — APPROVED** [review]

**Review by bama-nordic — DISMISSED** [review]
Should the firmware version be displayed everytime we boot or only when we explictly query? Need to finalize this.

**Comment by sachinthegreen** [comment]
> Should the firmware version be displayed everytime we boot or only when we explictly query? Need to finalize this.

Discussed with @bama-nordic and @udaynordic and it was agreed to display the version everytime we boot.

**Pull request merged**